components/src/tokens/border.ts

```diff
@@ -27,4 +27,7 @@ export const radii = {
   '3xLarge': '24px',
   '4xLarge': '40px',
   full: '9999px',
+  input: '0.5rem',
+  card: '1rem',
+  checkbox: '0.25rem',
```

Comment:
why are these custom values? also why `rem`?

Reply:
I was going for a more descriptive field. Something where you wouldn't need to look at the size token in order to know which one to choose.